synthpop/ipf/ipf.py

```diff
@@ -52,20 +52,16 @@ def calc_diff(x, y):
 
     iterations = 0
 
+    list_of_loc = (
```

Comment:
I think this should probably be a list comprehension instead of a generator expression. You can only iterate over generators once.